chcemvediet/apps/inforequests/tests/test_models/test_inforequest_draft.py

```diff
@@ -48,6 +51,15 @@ def test_content_field_default_value_if_omitted(self):
         draft = self._create_inforequest_draft(omit=[u'content'])
         self.assertEqual(draft.content, ())
 
+    def test_modified_field(self):
+        dt = local_datetime_from_local(u'2014-10-05 10:33:00')
+        draft = self._create_inforequest_draft(modified=dt)
```

Comment:
@martinmacko47 Ako budeme testovat field s `auto_add`? Teraz som chcel explicitne zadat nami zvoleny datum, aby bolo vidiet ze aj napriek tomu sa nastavy automaticky, po `save` metode.

Reply:
Ano, spravne. Treba otestovat, ze ak sa `modified` zada, tak sa odignoruje a aj tak sa savne aktualny cas. Akurat by sa zislo zmenit nazov testu nejak v zmysle, ze hodnota `modified` fieldu je ignorovana. Pozri ostatne testy, ci niekde uz nemame taky pripad, kde testujeme, ze nejaka hodnota sa pri vytvarani instancie ignoruje.

Reply:
podobny nazov je v `test_branch`. Je to sice trocha odlisne od vyznamu a logiky ze ignoruje hodnotu, ale skoro to iste.